cli-output/src/cli_output.rs

```diff
@@ -610,25 +610,56 @@ fn show_votes_and_credits(
         return Ok(());
     }
 
-    writeln!(f, "Recent Votes:")?;
-    for vote in votes {
-        writeln!(f, "- slot: {}", vote.slot)?;
-        writeln!(f, "  confirmation count: {}", vote.confirmation_count)?;
-    }
-    writeln!(f, "Epoch Voting History:")?;
     writeln!(
         f,
-        "* missed credits include slots unavailable to vote on due to delinquent leaders",
+        "{} Votes (using {}/{} entries):",
+        (if votes.len() < MAX_LOCKOUT_HISTORY {
+            "All"
+        } else {
+            "Recent"
+        }),
+        votes.len(),
+        MAX_LOCKOUT_HISTORY
     )?;
-    for entry in epoch_voting_history {
+
+    for vote in votes.iter().rev() {
+        writeln!(f, "- slot: {} (confirmation count: {})", vote.slot, vote.confirmation_count)?;
+    }
+    if let Some(newest) = epoch_voting_history.iter().rev().next() {
+        writeln!(
+            f,
+            "- ... (truncated older {} votes, which was rooted and became credits)",
```

Comment:
This serves two purposes:

- Actually indicate this list is truncated
- Also, as a logical cue for the relation of rooted votes <=> credits and popping-out nature

Reply:
Also, another word bike shedding:

`became` or `converted` or `translated to`? (I wanted to indicate the 1-by-1 nature of votes/credits)